webviz_subsurface/containers/_inplace_volumes_onebyone.py

```diff
@@ -139,6 +140,55 @@ def add_webvizstore(self):
             ),
         ]
 
+    @property
+    def tour_steps(self):
+        return [
+            {
+                "id": self.graph_id,
+                "content": "The chart shows inplace volumetrics results. ",
+            },
+            {
+                "id": self.table_id,
+                "content": "The table shows statistics per sensitivity. \
+                            Rows can be filtered by searching, and sorted by \
+                            clicking on a column header. ",
+            },
+            {
+                "id": self.response_id,
+                "content": "Select the volumetric calculation to display. ",
+            },
+            {
+                "id": self.plot_type_id,
+                "content": "Controls the type of the visualized chart. \
+                            Per realization shows bars per realization, \
+                            while the boxplot shows the range per sensitivity. ",
+            },
+            {
+                "id": self.tornadowrapper_id,
+                "content": "Tornado plot for the currently displayed data. \
+                            Differences references can be set and sensitivities \
+                            smaller than the reference can be filtered out. \
+                            Click on the bar of a sensitivity to highlight the \
+                            relevant realizations in the main chart.",
+            },
+            {
+                "id": self.ensemble_id,
+                "content": "If several ensembles are available, the active ensemble \
+                            can be selected here. ",
+            },
+            {
+                "id": self.source_id,
+                "content": "If volumes have been calculated for different grids. \
+                            E.g. geogrid and eclipsegrid, the active grid can be selected here. ",
```

Comment:
Maybe we could make an issue out of this: The source selector does not appear if only calculated for one grid?

Reply:
I think it should be there even for one grid. Nice to know which grid was used for the calculation.